lib/graph/digraph.nit

```diff
@@ -946,3 +946,133 @@ class HashDigraph[V: Object]
 
 	redef fun vertices_iterator: Iterator[V] do return outgoing_vertices_map.keys.iterator
 end
+
+# A reflexive directed graph
```

Comment:
Can you add more documentation about why this is useful and why it cannot be achieved with `HashDigraph` please?

Reply:
`HashDigraph` can manage the reflexive links but these must be explicitly indicated by the user. Creating another specific class for that (`ReflexiveHashDigraph`) avoids having to define the links manually as reflexive.